src/lib/y2network/interface_config_builder.rb

```diff
@@ -91,5 +87,70 @@ def load_sysconfig(devmap)
     def load_s390_config(devmap)
       @s390_config = !devmap.nil? ? @s390_config.merge(devmap) : @s390_config
     end
+
+  private
+
+    # returns a map with device options for newly created item
+    def init_mandatory_options
+      # FIXME: NetHwDetection is done in Lan.Read
+      Yast.import "NetHwDetection"
+
+      # #104494 - always write IPADDR+NETMASK, even empty
+      # #50955 omit computable fields
+      config["BROADCAST"] = ""
+      config["NETWORK"] = ""
+
+      if !config["NETMASK"] || config["NETMASK"].empty?
+        config["NETMASK"] = Yast::NetHwDetection.result["NETMASK"] || "255.255.255.0"
```

Comment:
how can fallback happen and is it nice behavior to user to write something that they do not specify?